subsys/net/lib/http/Kconfig

```diff
@@ -37,6 +37,52 @@ config HTTP_SERVER
 	  HTTP server support.
 	  Note: this is a work-in-progress
 
+#if HTTP_SERVER
+
+config NET_HTTP2_MAX_CLIENTS
+	int "Max number of HTTP/2 clients"
+	default 3
+	range 1 100
+	help
+	  This setting determines the maximum number of HTTP/2 clients that the server can handle at once.
+
+config NET_HTTP2_MAX_STREAMS
```

Comment:
```suggestion
config NET_HTTP_MAX_STREAMS
```

Reply:
I would even go further to say that Kconfig options specific to the server should have the prefix `NET_HTTP_SERVER_`.